config/routes.rb

```diff
@@ -1,10 +1,19 @@
 Rails.application.routes.draw do
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
+  get 'users/show'
```

Comment:
- [OPTIONAL]
- [ ] Since you are already using resources below to create the routes, you can take away the highlighted pieces of code as it is doing exactly the same thing